cnd/src/protocol_spawner.rs

```diff
@@ -49,6 +52,27 @@ impl ProtocolSpawner {
             halight_states,
         }
     }
+
+    pub fn supports_halight(&self) -> anyhow::Result<()> {
```

Comment:
This is done on line 110 already so we can delete it here.

Reply:
Done, thanks.